README.md

```diff
@@ -51,6 +51,28 @@ or valuable testing.
 
 Please read LICENSE.md, it is very simple and very liberal.
 
+Features
+--------
+
+* Zlib compatible API with support for dual-linking
```

Comment:
Maybe mention our zlib-ng native API aswell?
Something like "Modernized native API based on zlib api for ease of porting."?

Reply:
Added.